packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-embedded-widget/main/angular/next-steps.md

```diff
@@ -0,0 +1,3 @@
+You have now successfully authenticated with Okta! Now what? With a user's `id_token`, you have basic claims for the user's identity. You can extend the set of claims by modifying the `scopes` to retrieve custom information about the user. This includes `locale`, `address`, `groups`, and [more](/docs/reference/api/oidc/).
+
+Want to learn how to use the user's `access_token`? Check out our [how-to guide](/docs/guides/sign-into-spa/angular/main/) to learn about protecting routes on your server, validating the `access_token`, and more!
```

Comment:
```suggestion
Want to learn how to use the user's `access_token`? Check out our [how-to guide](/docs/guides/sign-into-spa/angular/main/) to learn about protecting routes on your server, validating the `access_token`, and more.
```

Reply:
I removed this section as the reference will no longer be accurate.